index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zuwairanajma Web</title>
+    <script src = "https://kit.fontawesome.com/1935d064dd.js" 
+    crossorigin = "anonymous">
+   </script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!-- Toolbar Header Section starts-->
+    <header> 
+        <div id= "head-wrap" class="headbar">
+    
+  <nav class = "headbar-menu">
+    <p class=""logo>My Logo</p>
+    <i class="fas fa-bars fa-lg"></i>
+  </nav>
+</div>
+</header>
+<!-- Toolbar Header Section ends -->
+
+<main>
+    <!-- Headline Section Start -->
+    <section class="headline">
+    <div class="container_text">
+        <h1 class="ptext">Hey There. <br/>I'm Zuwairanajma.</h1>
+        <h3 class="stext">I'm a Software Developer</h3>
+        <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
```

Comment:
- [ ] I highly suggest that you follow the Figma template. Currently, there are some minor problems with your design:
<table>
<tr>
<th>Your design</th>
<th>Figma</th>
</tr>
  <tr>
    <td><img src="https://github.com/Zuwairanajma/Portfolio-Projects/assets/86069740/22ae68ed-e0d3-4525-ab99-8c3730b1e39f"/></td>
    <td><img src="https://github.com/Zuwairanajma/Portfolio-Projects/assets/86069740/3188014d-bcc1-4d61-89b7-f3670fa29ad0"/></td>
  </tr>
</table>

- Use the correct fonts for the title, sub-title & paragraph.
- Make sure to pack all of the elements to match the Figma design.